modules/sagemaker/sagemaker-model-cicd/lib/code-pipeline/model-build-support/model-build-support-stack.ts

```diff
@@ -161,7 +161,7 @@ export class ModelBuildSupportStack extends cdk.Stack {
             'sagemaker:CreateModel',
           ],
           resources: [
-            `arn:aws:sagemaker:${cdk.Stack.of(this).region}:${cdk.Stack.of(this).account}:model-package/${modelPackageGroupName}/*`,
+            `arn:${cdk.Aws.PARTITION}:sagemaker:${cdk.Stack.of(this).region}:${cdk.Stack.of(this).account}:model-package/${modelPackageGroupName}/*`,
```

Comment:
Minor nitpick: We're using `cdk.Stack.of(this).region` to get region, but we're not using `cdk.Stack.of(this).partition` to get the partition, in the same line. I think we should be consistent, at least on a file or module level, so that we either use `cdk.Aws.*` everywhere, or `cdk.Stack.of(this).*` everywhere.

Reply:
Done. I changed them all although some of them have an environment that gets passed around instead of using the stack's region (I think this is for cross-account/region stuff, but I don't think that works cross-partition?)